packages/venia-concept/package.json

```diff
@@ -19,13 +19,14 @@
     "build:esm": "BABEL_ENV=development babel src --out-dir esm --root-mode 'upward' --source-maps --copy-files",
     "build:prod": "webpack -p --color --progress --profile --env.mode production",
     "clean": "rimraf dist esm",
+    "download-schema": "graphql get-schema --project venia",
     "prepublishOnly": "yarn run build",
     "start": "node server.js",
     "start:debug": "node --inspect-brk ./node_modules/.bin/webpack-dev-server --progress --color --env.mode development",
     "stats": "webpack-bundle-analyzer dist/build-stats.json",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "storybook:build": "build-storybook -c .storybook -o storybook-dist",
-    "validate-queries": "node ./validate-queries.js",
+    "validate-queries": "yarn run download-schema && graphql validate-queries --project venia --filesGlob \"src/**/*.{js,graphql,gql}\" --clients apollo literal && node ./summarize-validate-queries.js",
```

Comment:
Eventually we'd want the same command to both:

- Confirm that the cached local schema still matches the remote schema
- Validate all the queries

I figure once we have a real `@magento/pwa-buildpack` cli, like `pbuild`,  it would look a little more like:

```
"validate-queries": "pbuild verify-schema && graphql validate-queries | pbuild diagnose-queries"
```

Speaking of which, can you try making the CLI flags optional, and having them fall back to extension configuration in `.graphlconfig`?

Reply:
I ended up just removing them and always pulling them from `.graphqlconfig`.